processor/src/operations/io_ops.rs

```diff
@@ -523,21 +523,21 @@ mod tests {
 
     #[test]
     fn op_read() {
-        // reading from tape should push the value onto the stack
-        let mut process = Process::new_dummy_with_advice_tape(&[3]);
+        // reading from stack should push the value onto the stack
+        let mut process = Process::new_dummy_with_advice_stack(&[3]);
         process.execute_op(Operation::Push(ONE)).unwrap();
         process.execute_op(Operation::Read).unwrap();
         let expected = build_expected_stack(&[3, 1]);
         assert_eq!(expected, process.stack.trace_state());
 
-        // reading again should result in an error because advice tape is empty
+        // reading again should result in an error because advice stack is empty
         assert!(process.execute_op(Operation::Read).is_err());
     }
 
     #[test]
     fn op_readw() {
-        // reading from tape should overwrite top 4 values
-        let mut process = Process::new_dummy_with_advice_tape(&[3, 4, 5, 6]);
+        // reading from stack should overwrite top 4 values
```

Comment:
I'd change this to: "popping a word from the advice stack should overwrite top 4 elements of the operand stack"

Reply:
Done!